.vscode/settings.json

```diff
@@ -7,5 +7,10 @@
     "typescriptreact"
   ],
   "eslint.workingDirectories": ["./frontend-ssr"],
-  "editor.tabSize": 2
+  "editor.tabSize": 2,
+  "emmet.excludeLanguages": [
```

Comment:
Hmm, not sure we need those? Let's have a sync today and figure this indentation problem together :D